readme.txt

```diff
@@ -1,6 +1,6 @@
 === WooCommerce ===
 Contributors: automattic, mikejolley, jameskoster, claudiosanches, kloon, rodrigosprimo, peterfabian1000, vedjain, jamosova, obliviousharmony
-Tags: ecommerce, e-commerce, store, sales, sell, shop, cart, checkout, downloadable, downloads, payments, paypal, storefront, stripe, woo commerce, woo
+Tags: e-commerce, store, sales, sell, woo, shop, cart, checkout, downloadable, downloads, payments, paypal, storefront, stripe, woo commerce, woo
```

Comment:
Please note that only the first five tags will appear on the plugin page.

https://wordpress.org/plugins/woocommerce/

https://github.com/woocommerce/woocommerce/blob/fb1001b14a90b3020ca8000e424c955845142f5c/readme.txt#L3 

Visible tags: `e-commerce`, `ecommerce`, `sales`, `sell`, `store`

---

https://wordpress.org/plugins/jetpack/

https://github.com/Automattic/jetpack/blob/35c35c5a2b13446e68489210ee3edc5b12c8ed73/readme.txt#L3

Visible tags: `backup`, `jetpack`, `related posts`, `security`, `WordPress.com`

Reply:
Yes, I'm aware that only the first five tags are used by WP.org. That is why I added 'woo' as the fifth tag. I decided to leave the remaining tags as I don't know if there is another reason for them to exist as I don't know why they were added in the first place. Anyway thanks for the reminder.